src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -231,6 +240,23 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
         return true;
     }
 
+    protected UberallsClient getUberallsClient(Logger logger, String gitUrl, String branch) {
```

Comment:
This getter/setter exist so I can inject a mock uberalls client in test. I can add that functionality.